docs/Reference/Console/GetStarted/setup.md

```diff
@@ -0,0 +1,59 @@
+--8<-- "References/abbreviations.md"
+
+!!! warning "Warning - Console in Alpha Development"
+    Console is currently under development. The current available tags for download are Alpha version code. This could mean that certain features may not function yet, visual look and feel may change, or bugs/errors may occur. It is not recommended for use in Production deployments. Follow along our [Feature Backlog for future releases and feature updates](https://github.com/orgs/open-amt-cloud-toolkit/projects/10)
+
+## What You'll Need
+
+**Configure a network that includes:**
+
+-  A development system 
+-  An activated Intel AMT device
+
+<br>
+
+## What You'll Do
+
+<figure class="figure-image">
+  <img src="..\..\..\..\assets\images\Console_GetStarted_Alpha.png" alt="Figure 1: Get Started with Console">
+  <figcaption>Figure 1: Get Started with Console</figcaption>
+</figure>
+
+**To complete a deployment:**
+
+- Download and run Console.
+- Activate and configure an Intel AMT device.
+- Add a device to Console.
+- Manage a device using Console.
+
+## Get Console
+
+### Download
+
+1. Find the latest release of Console under [Github Releases](https://github.com/open-amt-cloud-toolkit/console/releases/latest).
+
+2. Download the appropriate binary assets for your OS and Architecture under the *Assets* dropdown section.
+
+3. Unzip the downloaded file.
+
+### Run
+
+1. Run the Console executable.
```

Comment:
Should we add a note saying that on Windows you might get a warning and that this is because our executable isn't signed yet during alpha phase?